android/app/src/androidTest/java/com/trustwallet/core/app/blockchains/ontology/TestOntologySigning.kt

```diff
@@ -0,0 +1,125 @@
+package com.trustwallet.core.app.blockchains.ontology
+
+import com.google.protobuf.ByteString
+import com.trustwallet.core.app.utils.Numeric
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import wallet.core.jni.OntologySigner
+import wallet.core.jni.proto.Ontology
+
+
+fun String.toByteString(): ByteString {
```

Comment:
Can you put this extension function in the utility class `Extensions.kt`, much better since it can be re-used later.

Reply:
okay, we will update it.